components/remote_receiver.rst

```diff
@@ -56,18 +56,22 @@ Configuration variables:
   - **sony**: Decode and dump Sony infrared codes.
   - **toshiba_ac**: Decode and dump Toshiba AC infrared codes.
 
-- **tolerance** (*Optional*, int): The percentage that the remote signal lengths can deviate in the
-  decoding process. Defaults to ``25%``.
+- **tolerance** (*Optional*, int or :ref:`config-time`): The percentage or time that the remote signal lengths can
+  deviate in the decoding process. Allowed values are in range ``0`` to ``4294967295%`` or ``0`` to ``4294967295us``. Defaults to
```

Comment:
Does that percentage make sense?
```suggestion
  deviate in the decoding process. Allowed values are in range ``0`` to ``100%`` or ``0`` to ``4294967295us``. Defaults to
```

Reply:
Sure, we can limit the value to 100% but in my opinion the percentage based tolerance doesn't make much sense at all since every protocol has different needs. Ideally the acceptable timing range should be defined by each protocol and not globally by the receiver.